examples/mockups/index.html

```diff
@@ -48,7 +48,7 @@
 
             <div class="column seven wide computer nine wide tablet sixteen wide mobile">
               <h1 class="ui header massive">
-                <span class="ui header">Continuous documentation for technical projects</span>
+                <span class="ui header">Push your docs, <br>we'll do the rest</span>
```

Comment:
The H1 in the old site is "Technical documentation lives here", which I quite like. "Continuous documentation for technical projects". This proposal is a little bit more emotional. What do you think?

Reply:
I like the idea of starting with the "push" action, associating the product to version control right from the first word.

I would also love to have feedback from the rest of the team here. What do you think? This is the first sentence the user will see. If you had 3 seconds to call someone's attention to Read the Docs, what would you say?

Reply:
> even though we could do more to be more technically correct or communicate what "the rest" is here. 

This is my main feedback, but I couldn't find a way to improve it with a similar number of words. I think the rest of the page will explain "the rest", so this seems fine for a first pass.

Reply:
What is "the rest"? :) To me, generally speaking, is "build and host". So, what about something that includes those words?

* "Push your docs, we will build and host them nicely"
* "Push your docs, we will host them nicely"

("nicely" is not the exact word that I want, but probably a synonymous of it)

Reply:
Seems we mostly agree on "the rest" being vague, perhaps it makes sense to continue on and iterate on this later?

To me, this is enough to work with and we can come back to this with more ideas. I actually think keeping the header short and vague works well, especially in combination with supporting elements to help define "the rest". If we want to continue on this path, I have some ideas for supporting elements which we could add later.

Reply:
Oh, yes! I don't think we have to block this work on this 👍🏼

Reply:
Cool, it seems we can continue with the suggestion from @astrojuanlu and I've opened https://github.com/readthedocs/site-community/issues/30 to come back to later.